packages/dev-server/index.js

```diff
@@ -76,5 +77,12 @@ module.exports = function() {
 
   // Start the server
   const port = process.env.PORT || 3000;
-  app.listen(port, announceServer);
+
+  const setupServer = new Promise(function(resolve, reject) {
+    resolve(app.listen(port, announceServer));
+  });
+
+  setupServer.then(open(`http://localhost:3000`));
+
+  // Doesn't wait for Webpack Bundle Analyzer to finalize before opening localhost.
```

Comment:
If this comment is accurate, and it’s finishing the Webpack bundle before starting, and just not finishing starting the Webpack Bundle Analyzer, I think that’s totally fine.

This is a good explanatory comment, so leave it in.